incubator/drupal/templates/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.image }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        env:
```

Comment:
Personally I prefer the separated tier approach too, but we've run into issues with configuring virtual hosts properly when the web server isn't in the same container as the application. Additionally, many of the applications we're configuring make use of `.htaccess` files, another reason it's easier to have the web server and the app in the same container, and also why nginx is a little harder to configure. That's why we've gone with the Apache + mod_php approach in the Bitnami application containers for now.